src/stores/cookieData.ts

```diff
@@ -0,0 +1,22 @@
+import { create } from 'zustand';
+
+export type CookieDataState = {
+  cookieData: boolean;
+  set: (state: Partial<CookieDataState>) => void;
+  checkCookieData: () => void;
+};
+
+type CookieDataStore = CookieDataState & {
+  set: (state: Partial<CookieDataState>) => void;
```

Comment:
@jackson-harris-iii why is it necessary to define `set` and `checkCookieData` in both the State and Store types?

Reply:
@charleslavon I was able to remove it from the state types.